src/config/yaml.ts

```diff
@@ -276,7 +276,21 @@ export const getTestAccounts = (config = yamlConfig): TestAccount[] =>
   }))
 
 export const getCronConfig = (config = yamlConfig): CronConfig => config.cronConfig
-export const getKratosConfig = (config = yamlConfig): KratosConfig => config.kratosConfig
+
+export const getKratosConfig = (config = yamlConfig): KratosConfig => {
+  let { publicApi, adminApi } = config.kratosConfig
+  if (process.env.KRATOS_PUBLIC_API) {
```

Comment:
Why do we need both `yaml` and `env` config options here?

Reply:
oh I think I did it before I was overriding value in `.env.ci` or `docker-compose`. I can probably remove it.